subsys/mgmt/mcumgr/Kconfig

```diff
@@ -277,6 +277,16 @@ config OS_MGMT_TASKSTAT_SIGNED_PRIORITY
 	  Enable this option only if your client software is able to properly decode
 	  and accept signed integers as priorities.
 
+if !THREAD_NAME
+
+config OS_MGMT_TASKSTAT_USE_THREAD_IDX_FOR_NAME
```

Comment:
I would create a chose of three values for configure what will be transferred under thread name.

Reply:
This is a little bit complicated because the usage of idx is default while THREAD_NAME is not enabled, but when you enable THREAD_NAME you may still want to use the idx. I was also considering to defaulting the option to y.

Reply:
Done